### PR TITLE
Fix/text field error and success text alignments

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

Fix text field error and success text alignments by specifying explicit line height

## Related Issue

https://helsinkisolutionoffice.atlassian.net/browse/HDS-703
https://github.com/City-of-Helsinki/helsinki-design-system/issues/467

Closes #

https://github.com/City-of-Helsinki/helsinki-design-system/issues/467

## How Has This Been Tested?

- Locally on developers machine
- Automated loki tests to ensure no breaking visual changes to the components itself

## Screenshots (if appropriate):

image

->

image

The issue can be reproduced by setting a line height of 15px to the whole component container. The error text (and success text) will inherit that line height which will break the vertical alignment.

### CODE REVIEW

**Review by laurakarhu — APPROVED** [review]
Well done!

File: packages/core/src/components/text-input/text-input.css [hunk 1]
```diff
@@ -130,6 +130,7 @@
   color: var(--helper-color-invalid);
   display: block;
   font-size: var(--fontsize-body-m);
+  line-height: var(--lineheight-l);
```
Comment (minevala) [inline packages/core/src/components/text-input/text-input.css:133]: Should we add line-height for helper-text too?

**Review by minevala — APPROVED** [review]
Good job!

File: packages/core/src/components/text-input/text-input.css (hunk 1 above)
Comment (VilleMiekkoja) [reply packages/core/src/components/text-input/text-input.css:133]: Good point. I'll add line-height for it too.

- [ ] Add line-height for helper-text too.

**Pull request merged**